src/main/scala/nl/knaw/dans/api/sword2/package.scala

```diff
@@ -43,6 +43,8 @@ package object sword2 {
 
   case class BagStoreSettings(baseDir: String, baseUrl: String)
 
+  case class PropertiesResources(bagId: String, filePaths: List[String])
```

Comment:
Do you mean just renaming the parameters, or also giving another type (URI)?
About the filePaths/fileUri's: I had understood that we don't give the complete file Uri's, but a path relative to the bagUri.